### PR TITLE
Document web-ext run --devtools command line argument

### PR DESCRIPTION
**Pull request by ochameau**
Document the new command line argument introduced in https://github.com/mozilla/web-ext/pull/2488

### CODE REVIEW

File: src/content/documentation/develop/web-ext-command-reference.md [hunk 1]
```diff
@@ -189,6 +189,18 @@ Environment variable: `$WEB_EXT_BROWSER_CONSOLE=true`
...
+#### `--devtools`
+
+This opens the Developer Tools for the installed extension on startup. See [this documentation](https://extensionworkshop.com/documentation/develop/debugging/) for more information. Example:
+
+```shell
+web-ext run --devtools
+```
+
+Note: The opened Developer Tools may not show all debugging output from content-scripts. Use the web console when debugging content-scripts.
+Note #2: This option only works with Firefox 106 and newer.
```
Comment (willdurand) [inline src/content/documentation/develop/web-ext-command-reference.md:202]: We might want to add the web-ext version as well

Comment (ochameau) [reply src/content/documentation/develop/web-ext-command-reference.md:202]: Thanks for the suggestion!

**Review by willdurand — APPROVED** [review]
Looks good, thanks.

**Comment by willdurand** [comment]
Let's wait until we release `web-ext` v7.3.0 to merge this PR.

**Pull request merged**